src/main/java/org/fundacionjala/at15/katas/pokerhands/mauricio/PokerHand.java

```diff
@@ -0,0 +1,73 @@
+package org.fundacionjala.at15.katas.pokerhands.mauricio;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PokerHand {
+    private List<Card> cards;
+
+    private final int numberCards = 5;
+    private final int score50ForFlush = 50;
+
+    public PokerHand(Card card1, Card card2, Card card3, Card card4, Card card5) {
+        cards = new ArrayList<>();
+        cards.add(card1);
+        cards.add(card2);
+        cards.add(card3);
+        cards.add(card4);
+        cards.add(card5);
+    }
+
+    public int getPokerHandScore() {
```

Comment:
I really like the way you distributed the logic, the code looks cleaner and easier to check behavior of different functionality, good job on the separation of methods!